dist/index.html

```diff
@@ -21,6 +21,7 @@ <h1>Travel Tracker</h1>
       <section class="trips">
         <section class="trip-choice-buttons">
           <button class="all-trips-button">ALL TRIPS</button>
+          <button class="past-trips-button">PAST TRIPS</button>
```

Comment:
Love the use of buttons here to see the respective trip data. I'm actually approaching it the exact same way.